save-cloud-charts/cpg/templates/demo-cpg-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo-cpg
```

Comment:
I actually thought that it can be simply included into the same chart as other services. There shouldn't be any problems with accessing the database from a different namespace

Reply:
I thought since we extracted neo4j to a dedicated chart, backend for neo4j also should be extracted